yang/frr-pim.yang

```diff
@@ -116,36 +116,32 @@ module frr-pim {
 
   grouping msdp-timers {
     leaf hold-time {
-      type uint32 {
-        range 3..600;
+      type uint16 {
+        range "1..max";
       }
       units seconds;
       default 75;
       description
         "Hold period is started at the MSDP peer connection establishment
          and is reset every new message. When the period expires the
-         connection is closed.
-
-         This value needs to be greater than `keep-alive-period`.";
+         connection is closed.";
```

Comment:
We could fix it to say: `This value needs to be greater than `keep-alive-period` in the remote peer.

Reply:
updated with note on remote